idx.coffee

```diff
@@ -3,6 +3,7 @@ Components =
 	Templatize: require('./src/Templatize')
 	Parameter: require('./src/Parameter')
 	Parameterize: require('./src/Parameterize')
+	Condition: require('./src/Parameter')
```

Comment:
```suggestion
	Condition: require('./src/Condition')
```

Reply:
fixed in [c051e58](https://github.com/UnquietCode/Cloud-Temple/pull/2/commits/c051e58ef5d135cb17c4cd6f9f27c1f877a9c6c6)